maps/torch/torch3_deck3.dmm

```diff
@@ -14779,12 +14797,32 @@
 	},
 /turf/simulated/floor/plating,
 /area/maintenance/thirddeck/port)
+"DE" = (
+/obj/machinery/power/apc{
+	dir = 4;
+	name = "east bump";
+	pixel_x = 24
+	},
+/obj/structure/cable/green{
+	d2 = 2;
+	icon_state = "0-2"
+	},
+/obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,
+/obj/machinery/atmospherics/pipe/simple/hidden/blue{
+	dir = 10;
+	icon_state = "intact"
+	},
```

Comment:
Same with this one, it's leaking.

`## UNIT_TEST ##: [MAP: Station pipes shall not leak]: Following pipe is leaking: The pipe (/obj/machinery/atmospherics/pipe/simple/hidden/blue) ([0x2003b0d]) (the deck) (53,97,3) (/turf/simulated/floor/tiled/steel_ridged)`